tests/queries/0_stateless/00538_datediff.sql

```diff
@@ -74,3 +74,14 @@ SELECT dateDiff('second', toDateTime('2014-10-26 00:00:00', 'UTC'), toDateTime('
 SELECT 'Additional test';
```

Comment:
Btw, this suggests that you also needed to add a similar statement before the new test case.